pkg/sparrow/controller_test.go

```diff
@@ -236,79 +231,60 @@ func TestChecksController_Reconcile(t *testing.T) {
 
 func TestChecksController_RegisterCheck(t *testing.T) {
 	tests := []struct {
-		name    string
-		setup   func(t *testing.T) *ChecksController
-		check   checks.Check
-		wantErr bool
+		name  string
+		setup func() *ChecksController
+		check checks.Check
 	}{
 		{
 			name: "valid check",
-			setup: func(_ *testing.T) *ChecksController {
+			setup: func() *ChecksController {
 				return NewChecksController(db.NewInMemory(), NewMetrics())
 			},
-			check:   health.NewCheck(),
-			wantErr: false,
+			check: health.NewCheck(),
 		},
 		{
 			name: "duplicate check registration",
-			setup: func(t *testing.T) *ChecksController {
+			setup: func() *ChecksController {
 				cc := NewChecksController(db.NewInMemory(), NewMetrics())
 				check := health.NewCheck()
-				err := cc.RegisterCheck(context.Background(), check)
-				if err != nil {
-					t.Fatalf("RegisterCheck() error = %v", err)
-				}
+				cc.RegisterCheck(context.Background(), check)
 				return cc
 			},
-			check:   health.NewCheck(),
-			wantErr: false,
+			check: health.NewCheck(),
 		},
 	}
 
 	for _, tt := range tests {
 		t.Run(tt.name, func(t *testing.T) {
-			cc := tt.setup(t)
+			cc := tt.setup()
 
-			err := cc.RegisterCheck(context.Background(), tt.check)
-			if (err != nil) != tt.wantErr {
-				t.Errorf("RegisterCheck() error = %v, wantErr %v", err, tt.wantErr)
-			}
+			cc.RegisterCheck(context.Background(), tt.check)
```

Comment:
We should check for something in this test, otherwise we only test if it runs succussfully

Reply:
@lvlcn-t The scope of this PR is to remove the error from the controller. Let's not change the tests for now (which are fragile at best). We can open an issue regarding the tests.

Reply:
Yeah, let's open an issue.

Reply:
https://github.com/caas-team/sparrow/issues/131 done, will post PR soon to fix